charts/kyuubi/templates/NOTES.txt

```diff
@@ -30,9 +30,13 @@ In order to check the release status, use:
 {{ $name | snakecase | upper }}:
 - To access {{ $.Release.Name }}-{{ $name | kebabcase }} service within the cluster, use the following URL:
     {{ $.Release.Name }}-{{ $name | kebabcase }}.{{ $.Release.Namespace }}.svc.cluster.local
+{{- if $.Values.kyuubiConf.kyuubiDefaults }}
+{{- if or (contains "kyuubi.frontend.bind.host=localhost" $.Values.kyuubiConf.kyuubiDefaults ) (contains "kyuubi.frontend.bind.host=127.0.0.1" $.Values.kyuubiConf.kyuubiDefaults) }}
```

Comment:
should be ok, even it allows
- `kyuubi.frontend.bind.host <any spaces> localhost`
- `kyuubi.frontend.bind.host <any spaces> = <any spaces> localhost`

Reply:
Got it.
I updated the Helm template to use a single regexMatch function to match localhost or 127.0.0.1 with any number of spaces.